python-checks/src/main/java/org/sonar/python/checks/DeadStoreCheck.java

```diff
@@ -76,7 +84,9 @@ public void initialize(Context context) {
   private static void verifyBlock(SubscriptionContext ctx, CfgBlock block, LiveVariablesAnalysis.LiveVariables blockLiveVariables,
     Set<Symbol> readSymbols, FunctionDef functionDef) {
 
-    DeadStoreUtils.findUnnecessaryAssignments(block, blockLiveVariables, functionDef)
+    var unnecessaryAssignments = DeadStoreUtils.findUnnecessaryAssignments(block,
+      blockLiveVariables, functionDef);
+    unnecessaryAssignments
```

Comment:
Do we need this change? I don't have the feeling this makes the overall code clearer, and we don't use the `unnecessaryAssignments` variable anywhere else.